File: PL/scope/Scope.js
let i = 100;
let iarr = [1, 2, 3];
let s2 = '';
let j = 99;
addj(iarr);
for (let j = 0; j < iarr.length; ++j) {
  let s1 = 'Hello';
  s2 = s1;
  i = j;
  console.log(iarr[j]);
}
s1 = 'World';
console.log(s1, s2);
console.log('i=', i);

j = 0;
function addj(arr) {
  console.log('j=', j);
  for (let i = 0; i < arr.length; ++i) {
    arr[i] += j;
  }
}

File: PL/scope/Scope.py
j = 0


def addj(arr):
    global j
    print('j=', j)
    for i in range(len(arr)):
        arr[i] += j


i = 100
iarr = [1, 2, 3]
s2 = ""
j = 99
addj(iarr)
for j in range(len(iarr)):
    s1 = "Hello"
    s2 = s1
    i = j
    print(iarr[j])

s1 = "World"
print(s1, s2)
print("i=", i)
print("j=", j)
